README.md

```diff
@@ -302,6 +302,28 @@ hubspot.campaigns.events(opts, cb)
 hubspot.broadcasts.get(opts, cb)
 ```
 
+### Timeline
+
+```javascript
+hubspot.timelines.createEventType(applicationId, userId, data, cb)
+hubspot.timelines.updateEventType(eventTypeId, applicationId, data, cb)
+hubspot.timelines.createEventTypeProperty(
+  eventTypeId,
+  applicationId,
+  userId,
+  data,
+  cb,
+)
+hubspot.timelines.updateEventTypeProperty(
+  eventTypeId,
+  applicationId,
+  propertyId,
+  data,
+  cb,
+)
+hubspot.timelines.createTimelineEvent(applicationId, eventTypeId, data, cb)
```

Comment:
Two thoughts from discussing this in person:
1) Let's make the position of the `applicationId` argument consistent across the timeline functions
2) Consider adding a note that highlights where the API docs state that creating a timeline event returns a 204